src/xb-builder-node.c

```diff
@@ -297,6 +301,41 @@ xb_builder_node_parse_literal_text (XbBuilderNode *self, const gchar *text, gssi
 	return g_string_free (tmp, FALSE);
 }
 
+/**
+ * xb_builder_node_tokenize_text:
+ * @self: a #XbBuilderNode
+ *
+ * Tokenize text added with xb_builder_node_set_text().
```

Comment:
Would be helpful to document what ‘tokenizing’ means.

Perhaps would be useful to add that as a paragraph or two (“nodes can be tokenized. This means blah blah and allows for blah blah with 2x better performance at a cost of 2x more storage”) in the section-level documentation for `XbBuilderNode`, and then link to it from here (and the other new method below).

Reply:
I've added something.